JavaAssignment/src/rahulM/Calculator.java

```diff
@@ -0,0 +1,53 @@
+package switchcase;
+
+import java.util.Scanner;
+
+public class Calculator {
+	
+	public static void main(String[] args) {
+		Calculator calc =new Calculator();
+		calc.calC();
+	}
+
+    	
+		void calC(){
+		double num1, num2;
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Here, always try write parameter method so other class or user can also use your method.
nad try to not write scanner class in function method.